packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -0,0 +1,19 @@
+import { makeStyles } from '@fluentui/react-make-styles';
+import { KEYBOARD_NAV_ATTRIBUTE } from '../symbols';
+
+const useStyles = makeStyles({
+  focus: theme => ({
+    outline: 'none',
+    [`:global([${KEYBOARD_NAV_ATTRIBUTE}="true"])`]: {
+      ':focus': {
+        outline: `solid 1px red`,
```

Comment:
I'm guessing we're waiting on design for the actual focus indicator styles?

Reply:
No, that's the focus style!
Joking 😂, It's just like that for testing at the moment, I'll update it as soon as this is ready for review